Bottom: e046a0917dbcdabceab41e7ba3f82490a8b7bf17
Top:    206218e7d615048485b790000210276d87523fe3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-17 15:15:14 +0100

Refresh of bikeshed-commit-11700ba8c3c951

---

diff --git a/src/main.c b/src/main.c
index 270aae3..ddd5675 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1123,7 +1123,7 @@ Defaults to @code{"QueryLibrary.sh '%p' '%f' %a"}
 @item --lib-contents-command <string>
 Command to query the contents of the library. @*
 Defaults to @code{"ListLibraryContents.sh %p %f"} or,
-on Windows builds, an empty string.
+on Windows builds, an empty string (to disable this feature).
 @end ftable
 %end-doc
 */
